<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Vishnus-MacBook-Air.local" failures="2" tests="6" name="com.qa.tests.LoginTests" time="147.701" errors="0" timestamp="2021-12-31T14:46:17 IST" skipped="0">
  <testcase classname="com.qa.tests.LoginTests" name="successfulLogin" time="8.927"/>
  <system-out/>
  <testcase classname="com.qa.tests.LoginTests" name="invalidUserName" time="42.850">
    <failure message="expected [Username and password do not match any user in this service.] but found [Username and password do not match any user in this service.__test]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Username and password do not match any user in this service.] but found [Username and password do not match any user in this service.__test]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.tests.LoginTests.invalidUserName(LoginTests.java:87)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- invalidUserName -->
  <system-out>
    <![CDATA[This is the sample screenshot
<a href='/Users/vishnu/eclipse-workspace/TDDFramework/Screenshots/iOS_iPhone 13/2021-12-31-14-43-55/LoginTests/invalidUserName.png'> <img src='/Users/vishnu/eclipse-workspace/TDDFramework/Screenshots/iOS_iPhone 13/2021-12-31-14-43-55/LoginTests/invalidUserName.png' height='400' width='400'/> </a>]]>
  </system-out>
  <testcase classname="com.qa.tests.LoginTests" name="invalidUserName" time="9.571">
    <failure message="expected [Username and password do not match any user in this service.] but found [Username and password do not match any user in this service.__test]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Username and password do not match any user in this service.] but found [Username and password do not match any user in this service.__test]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.tests.LoginTests.invalidUserName(LoginTests.java:87)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- invalidUserName -->
  <system-out>
    <![CDATA[This is the sample screenshot
<a href='/Users/vishnu/eclipse-workspace/TDDFramework/Screenshots/Android_motorola one/2021-12-31-14-43-55/LoginTests/invalidUserName.png'> <img src='/Users/vishnu/eclipse-workspace/TDDFramework/Screenshots/Android_motorola one/2021-12-31-14-43-55/LoginTests/invalidUserName.png' height='400' width='400'/> </a>]]>
  </system-out>
  <testcase classname="com.qa.tests.LoginTests" name="successfulLogin" time="43.525"/>
  <system-out/>
  <testcase classname="com.qa.tests.LoginTests" name="invalidPassword" time="32.674"/>
  <system-out/>
  <testcase classname="com.qa.tests.LoginTests" name="invalidPassword" time="10.154"/>
  <system-out/>
</testsuite> <!-- com.qa.tests.LoginTests -->
